app/views/candidate_interface/apply_from_find/apply_on_ucas_only.html.erb

```diff
@@ -5,11 +5,14 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render CandidateInterface::UCASDowntimeComponent.new  %>
 
-    <h1 class="govuk-heading-xl">
-      <span class="govuk-caption-xl"><%= @course.name_and_code %></span>
-      <%= t('page_titles.apply_from_find') %>
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"><%= @course.name_and_code %></span>
+      <%= t('page_titles.apply_from_find_with_ucas') %>
```

Comment:
Ultimately we’ll want to adjust page heading sizes down from `xl` to `l`, but I think we should do that in one go in a separate PR.

```suggestion
    <h1 class="govuk-heading-xl">
      <span class="govuk-caption-xl"><%= @course.name_and_code %></span>
      <%= t('page_titles.apply_from_find_with_ucas') %>
```